docs/conf.py

```diff
@@ -440,5 +440,5 @@ def make_events():
 def setup(app):
     from xonsh.pyghooks import XonshConsoleLexer
 
-    app.add_lexer("xonshcon", XonshConsoleLexer)
+    app.add_lexer("xonshcon", XonshConsoleLexer())
```

Comment:
Doesn't this need to be a class, not an instance?

Reply:
Or is it some weird magic thing that Pygments does?

Reply:
It's what I needed to do to get Sphinx to stop complaining about code-block xonshcon.

Reply:
Ahh cool that's great!